### PR TITLE
Feat/layout refactor

### PR DESCRIPTION
**Pull request by Lamperoyge**
Refactored layout. Deleted unused sidebar component in other pages. Restructured _app.tsx 

https://www.loom.com/share/c579c9bb4ea1437581bb4861aa5aeb78

500 page:

Screenshot 2022-07-14 at 13 43 18

### CODE REVIEW

File: wondrous-app/utils/constants.ts [hunk 1]
```diff
@@ -514,3 +514,26 @@ export const PROPOSAL_VOTE_LABELS = {
     ACTION: 'Reject',
   },
 };
+
+export const PAGES_WITH_NO_SIDEBAR = [
```
Comment (terryli0095) [inline wondrous-app/utils/constants.ts:518]: nice

Comment (terryli0095) [reply wondrous-app/utils/constants.ts:518]: gonna set up a wiki/notion doc that reminds to add to this list

Comment (Lamperoyge) [reply wondrous-app/utils/constants.ts:518]: nice!

**Comment by terryli0095** [comment]
@andywong418  since it's more refactor code can you take a look?

**Review by andywong418 — APPROVED** [review]
Looks good!

**Pull request merged**